stonesoup/types/state.py

```diff
@@ -185,6 +187,48 @@ class WeightedGaussianState(GaussianState):
     """
     weight: Probability = Property(default=0, doc="Weight of the Gaussian State.")
 
+    @property
+    def gaussian_state(self):
+        """The Gaussian state."""
+        return GaussianState(self.state_vector,
+                             self.covar,
+                             timestamp=self.timestamp)
+
+    @classmethod
+    def from_gaussian_state(cls, gaussian_state, *args, copy=True, **kwargs):
+        """
+        Returns a WeightedGaussianState instance based on the gaussian_state.
+
+        Parameters
+        ----------
+        gaussian_state : TYPE
+            DESCRIPTION.
+        *args : See main :class:`~.WeightedGaussianState`
```

Comment:
`*` needs to be escaped `\*`

Reply:
For the documentation would you prefer the parameters to be listed as just "args" and "kwargs" rather than "*args" and "**kwargs", or would you prefer them with the "*"s and then escaped?

Reply:
I think it would be good to have `*args` and `**kwargs`.

Reply:
So, if I escape them then the Sphinx documentation works out, but pyflake complains that there is an invalid escape sequence '\\*'  - Oh the joy :) Do we just want to live with the lint error?

Reply:
Ah, that'll be because you need to escape the escape `\\*`. You need to escape the Python string, so it can get to the RST parser.

Reply:
Ok. I used an r string instead of escaping the escape. It's a bit cleaner. I pushed the changes. Let me know if there are any further issues.